Bottom: 2ba59e8b1c3f3b47b62f486eae93855c8d3403d2
Top:    7b4a3c0ff04ee29dc7ca13f0d53b2948fb2b99e9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-28 00:48:03 +0100

Refresh of give-the-hids-control-over-att

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 1ead6a9..45042d0 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -784,6 +784,7 @@ ghid_notify_crosshair_change (bool changes_complete)
     CrosshairOn ();
   else
     CrosshairOff ();
+  ghid_draw_area_update (gport, NULL);
 }
 
 static void
